docs/configuration.md

```diff
@@ -365,7 +365,7 @@ Apart from these, the following properties are also available, and may be useful
   <td>
     Extra classpath entries to prepend to the classpath of executors. This exists primarily for
     backwards-compatibility with older versions of Spark. Users typically should not need to set
-    this option.
+    this option. Also see <code>spark.executor.userClassPathFirst</code> to prepend jar ahead of classpath of spark.
```

Comment:
I don't think this clarifies; it doesn't cause these classpath entries (not jars) to be prepended, but to take precedence over Spark's. That's already stated in the docs though.